assets/src/blocks/PostsList/PostsListBlock.js

```diff
@@ -0,0 +1,71 @@
+const POSTS_LIST_BLOCK = 'planet4-blocks/posts-list-block';
+
+export const registerPostsListBlock = () => {
+  const {registerBlockVariation} = wp.blocks;
+  const {__} = wp.i18n;
+
+  return registerBlockVariation('core/query', {
+    name: POSTS_LIST_BLOCK,
+    title: 'Posts List',
+    icon: 'list-view',
+    description: __('Posts List is the place in Planet 4 that the latest articles, press releases and publications can be found.', 'planet4-blocks-backend'),
+    category: 'planet4-blocks-beta',
+    isActive: ({namespace, query}) => {
+      return (
+        namespace === POSTS_LIST_BLOCK &&
+              query.postType === 'post'
+      );
+    },
+    attributes: {
+      namespace: POSTS_LIST_BLOCK,
+      className: 'posts-list-query',
+      query: {
+        perPage: 3,
+        pages: 0,
+        offset: 0,
+        postType: 'post',
+        order: 'desc',
+        orderBy: 'date',
+        author: '',
+        search: '',
+        exclude: [],
+        sticky: '',
+        inherit: false,
+      },
+    },
+    scope: ['inserter'],
+    allowedControls: ['taxQuery'],
+    innerBlocks: [
+      ['core/heading', {content: __('Related Posts', 'planet4-blocks-backend')}],
+      ['core/paragraph', {
+        placeholder: __('Enter description', 'planet4-blocks-backend'),
+        style: {
+          spacing: {
+            margin: {
+              top: '24px',
+              bottom: '36px',
+            },
+          },
+        },
+      }],
+      ['core/post-template', {}, [
+        ['core/columns', {}, [
+          ['core/post-featured-image', {isLink: true, linkTarget: '_blank'}],
+          ['core/group', {}, [
+            ['core/post-terms', {
+              term: 'category',
+              separator: ' ',
+            }],
+            ['core/post-title', {isLink: true, linkTarget: '_blank'}],
+            ['core/post-excerpt'],
+            ['core/group', {className: 'posts-list-meta'}, [
+              ['core/post-author-name', {isLink: true, linkTarget: '_blank'}],
+              ['core/post-date'],
```

Comment:
Do we need to add the reading time after this, similar to what we do for [listing pages](https://github.com/greenpeace/planet4-master-theme/blob/05346596a9e9255c5df800429c559550b68d6800/parts/query-grid.html#L29)?

Reply:
Hey @mleray and @comzeradd, so I was looking into this and after a conversation with @lithrel, this [block](https://github.com/greenpeace/planet4-master-theme/blob/main/functions.php#L237) is not registered on the frontend so it throws an error and crashed the editor. We could create another ticket to register the block to the frontend and add to the Posts List block(so as to maintain this ticket's scope) or do we do it in this ticket?

Reply:
I can create a follow up ticket for the "reading time" block to be added in the Posts List